openproblems/tasks/_batch_integration/batch_integration_feature/README.md

```diff
@@ -45,6 +45,9 @@ Datasets should contain the following attributes:
 * `adata.layers['counts']` with raw, integer UMI count data,
 * `adata.layers['log_normalized']` with log-normalized data and
 * `adata.X` with log-normalized data
+* `adata.uns['n_genes_pre']` with the number of genes present before integration
+* `adata.uns['hvg_unint']` with a list of 2000 highly variable genes
```

Comment:
If you're requiring it be 2000 genes, you should also set this as the default in `precomp_hvg` and check it in `check_dataset`